src/pykeen/models/predict.py

```diff
@@ -115,6 +114,143 @@ def _get_input_batch(
     return target, batch, (batch_ids[0], batch_ids[1])
 
 
+def _get_mapped_triples(mapped_triples: Union[CoreTriplesFactory, MappedTriples]):
+    """Get mapped triples."""
+    if isinstance(mapped_triples, CoreTriplesFactory):
+        mapped_triples = mapped_triples.mapped_triples
+    return mapped_triples
+
+
+class PredictionPostProcessor:
+    """A post-processor for predictions."""
+
+    def __init__(self, **filter_triples: Union[None, CoreTriplesFactory, MappedTriples]) -> None:
+        """Instantiate the processor.
+
+        :param filter_triples:
+            the filter triples
```

Comment:
explicit about what keys are expected here (maybe better to be explicit with the kwargs too?)